gateway/Cargo.toml

```
@@ -0,0 +1,47 @@
+[package]
+name = "ipc-gateway"
+description = "Subnet Coordinator Actor (SCA) for IPC"
+version = "0.0.1"
+license = "MIT OR Apache-2.0"
+authors = ["ConsensusLab", "Protocol Labs", "Filecoin Core Devs"]
+edition = "2021"
+repository = "https://github.com/consensus-shipyard/ipc-gateway"
```

The name of the repo has changed. Be sure to change it in all `Cargo.toml`
```suggestion
repository = "https://github.com/consensus-shipyard/ipc-actor"
```